Modules/Login/Http/Services/LoginService.php

```diff
@@ -3,43 +3,31 @@
 namespace Modules\Login\Http\Services;
 
 use Illuminate\Support\Facades\Auth;
+use Modules\Login\Entities\User;
 
 class LoginService
 {
     /**
      * Sign In a user
      *
      * @param array $requestData
-     * @return array
+     * @return string
      */
-    public function attemptLogin(array $requestData): array
+    public function attemptLogin(array $requestData): string
     {
+        $data = '';
         if (Auth::attempt(["email" => $requestData['email'], "password" => $requestData['password']])) {
             $user = Auth::user();
             $role = $user->role;
-            if ($role == 'brand' || $role == 'retailer') {
+            if ($role == User::ROLE_BRAND || $role == User::ROLE_RETAILER) {
                 Auth::logout();
-                return [
-                    'res' => false,
-                    'msg' => 'Invalid credentials',
-                    'data' => ''
-                ];
+                return $data;
```

Comment:
instead of returning data, use true false only. we can check with true false condition. If user is right one then send true and if not then send false.
You are not using $user variable so there is no need to return it.

Reply:
fixed